sdk/eventgrid/azure-eventgrid/azure/eventgrid/_helpers.py

```diff
@@ -82,7 +86,7 @@ def _get_authentication_policy(credential):
     return authentication_policy
 
 def _is_cloud_event(event):
-    # type: dict -> bool
+    # type: (dict) -> bool
```

Comment:
Input is actually Any, since since if the input is not a dict, you don't raise an exception, you return False.

This will remove the cast later down the road

Reply:
MY rationale was that we expect only a dictionary here - but yes, you are right - changing it